packages/target-nodejs-sdk/src/api/delivery.js

```diff
@@ -0,0 +1,75 @@
+import { DECISIONING_METHOD, isDefined } from "@adobe/target-tools";
+import {
+  Configuration as DeliveryConfiguration,
+  DeliveryApi
+} from "@adobe/target-tools/delivery-api-client";
+import { executeSendBeacon } from "../utils";
+
+DeliveryApi.prototype.decisioningMethod = DECISIONING_METHOD.SERVER_SIDE;
+
+/**
+ *
+ * @returns { import("@adobe/target-tools/delivery-api-client/runtime").Configuration }
+ */
+export function createDeliveryConfiguration({
+  basePath,
+  fetchApi,
+  headers,
+  timeout
+}) {
+  return new DeliveryConfiguration({
+    basePath,
+    fetchApi,
+    headers,
+    timeout
+  });
+}
+
+/**
+ *
+ * @param { import("@adobe/target-tools/delivery-api-client/runtime").Configuration } configuration
+ * @returns { import("@adobe/target-tools/delivery-api-client").DeliveryApi }
+ */
+export function createDeliveryApi(configuration) {
+  const deliveryApi = new DeliveryApi(configuration);
+  return {
+    execute: (imsOrgId, sessionId, deliveryRequest, atjsVersion) =>
+      deliveryApi.execute(imsOrgId, sessionId, deliveryRequest, atjsVersion),
+    decisioningMethod: DECISIONING_METHOD.SERVER_SIDE
+  };
+}
+
+/**
+ *
+ * @param { import("@adobe/target-tools/delivery-api-client/runtime").Configuration } configuration
+ * @returns {GenericApi}
+ */
+export function createBeaconDeliveryApi(configuration) {
+  return {
+    execute: (organizationId, sessionId, deliveryRequest, atjsVersion) => {
+      const query = {
+        imsOrgId: organizationId,
+        sessionId
+      };
+
+      if (isDefined(configuration.version)) {
```

Comment:
as mentioned above, let's remove version altogether, as SDK name and version will be sent in `X-EXC-SDK*` headers